transforms/universal/doc_id/ray/pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[project]
+name = "dpk_docid_transform_ray"
+version = "0.4.0.dev6"
+requires-python = ">=3.10"
```

Comment:
should we use ">=3.10,<3.12" ?

Reply:
Technically yes, but we are doing this way everywhere else

Reply:
Is this a transitive dependency from data-prep-toolkit and if so, can it be removed?